docs/posts/execute_commands_on_host.md

```diff
@@ -91,6 +93,28 @@ if [ -n "${ZSH_VERSION-}" ]; then
 fi
 ```
 
+### fish
+
+Place this in a new fish config (such as `~/.config/fish/conf.d/cnf_handle.fish`:
+
+```fish
+function fish_command_not_found
+ # don't run if not in a container
+ if test ! -e /run/.containerenv -a ! -e /.dockerenv
```

Comment:
I think this function is a little bit more readable if you:
* reversed the logic of the container check
* used more idiomatic `fish`, such as `or` and `command -q`

I think the `exit 127` should be `return 127` since it is a function.

```fish
function fish_command_not_found
    # "In a container" check
    if test -e /run/.containerenv; or test -e /.dockerenv
        if command -q flatpak-spawn
            flatpak-spawn --host $argv
        else if command -q host-exec
            host-exec "$argv"
        else
            return 127
        end
    else
        __fish_default_command_not_found_handler "$argv"
    end
end
```

Reply:
Does `return 127` work? I don't have my Linux laptop nearby so I can't confirm.

I was about to change `exit 127` to `__fish_default_command_not_found_handler "$argv"` because yesterday I found that `exit 127` would cause shell to exit directly.

This function is mostly based on the original function for bash/zsh, I only modified it so it can run on fish, didn't notice the readability issue. I think you can modify my PR right?